modules/core/src/main/java/org/apache/ignite/internal/visor/consistency/VisorConsistencyRepairTask.java

```diff
@@ -89,10 +96,41 @@ protected VisorConsistencyRepairJob(VisorConsistencyRepairTaskArg arg, boolean d
 
         /** {@inheritDoc} */
         @Override protected String run(VisorConsistencyRepairTaskArg arg) throws IgniteException {
+            ExecutorService sys = ignite.context().pools().getSystemExecutorService();
+
+            Map<Boolean, List<IgniteBiTuple<Integer, String>>> res = arg.parts().stream()
+                .map(p -> F.t(p, sys.submit(() -> processPartition(p, arg))))
```

Comment:
May this cause system pool deadlock?
More safer way is to start some processes which will processes some queue's values one-by-one in parallel.
Let's make limiter = 8 threads by default.

Reply:
What scenario do you have in mind? Job requests executed on `ignite.context().pools().getExecutorService()`
You can see it from `GridJobProcessor#executeAsync` line 1569.

Reply:
> What scenario do you have in mind?
Repair may fill the system pool completely for a long time.
This may cause deadlocks ot timeouts.

Am I missed something?